src/components/common/TmSessionImportName.vue

```diff
@@ -94,16 +94,29 @@ export default {
       set(value) {
         this.$store.commit(`updateField`, { field: `name`, value })
       }
+    },
+    prefix: {
```

Comment:
nope, the prefix should be got from the networkId. please check how the createKey function in keystore.js works